client/my-sites/media-library/list-no-content.jsx

```diff
@@ -1,57 +1,57 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React from 'react';
 
 /**
  * Internal dependencies
  */
-var EmptyContent = require( 'components/empty-content' ),
-	UploadButton = require( './upload-button' );
+import EmptyContent from 'components/empty-content';
+import UploadButton from './upload-button';
+import { userCan } from 'lib/site/utils';
 
-module.exports = React.createClass( {
+export default React.createClass( {
 	displayName: 'MediaLibraryListNoContent',
 
 	propTypes: {
 		site: React.PropTypes.object,
 		filter: React.PropTypes.string
 	},
 
-	getLabel: function() {
-		var label;
-
+	getLabel() {
 		switch ( this.props.filter ) {
 			case 'images':
-				label = this.translate( 'You don\'t have any images.', { textOnly: true, context: 'Media no results' } );
-				break;
+				return this.translate( 'You don\'t have any images.', { textOnly: true, context: 'Media no results' } );
+
 			case 'videos':
-				label = this.translate( 'You don\'t have any videos.', { textOnly: true, context: 'Media no results' } );
-				break;
+				return this.translate( 'You don\'t have any videos.', { textOnly: true, context: 'Media no results' } );
+
 			case 'audio':
-				label = this.translate( 'You don\'t have any audio files.', { textOnly: true, context: 'Media no results' } );
-				break;
+				return this.translate( 'You don\'t have any audio files.', { textOnly: true, context: 'Media no results' } );
+
 			case 'documents':
-				label = this.translate( 'You don\'t have any documents.', { textOnly: true, context: 'Media no results' } );
-				break;
+				return this.translate( 'You don\'t have any documents.', { textOnly: true, context: 'Media no results' } );
+
 			default:
-				label = this.translate( 'You don\'t have any media.', { textOnly: true, context: 'Media no results' } );
-				break;
+				return this.translate( 'You don\'t have any media.', { textOnly: true, context: 'Media no results' } );
 		}
-
-		return label;
 	},
 
-	render: function() {
-		var action = (
-			<UploadButton className="button is-primary" site={ this.props.site }>
-				{ this.translate( 'Upload Media' ) }
-			</UploadButton>
-		);
+	render() {
+		let line, action;
```

Comment:
>It looks strange to me to not initialize these variables

At its base, I guess this is an argument of semantic differences between `undefined` and `null`. In my mind, passing `undefined` as the `action` and `line` prop to `<EmptyContent />` is just as meaningful as passing `null` values (lack of prop vs. explicitly empty prop), and has the advantage of being more concise.